src/ontology/uberon-edit.obo

```diff
@@ -117782,12 +117783,14 @@ property_value: external_ontology_notes "in FMA the Adbuctor muscle is a subclas
 id: UBERON:0006846
 name: surface groove
 def: "A furrow or an incomplete tube." [AEO:0000161, AEO:JB, https://github.com/obophenotype/human-developmental-anatomy-ontology/issues/4]
+comment: Surface groove are modelled as immaterial - see https://github.com/obophenotype/uberon/issues/2667 for discussion
```

Comment:
"Surface grooves are modelled as immaterial."
or
"Surface groove is modelled as immaterial."